mysite/fsm/models.py

```diff
@@ -152,9 +152,9 @@ class FSM(models.Model):
     """
     name = models.CharField(max_length=64, unique=True)
     title = models.CharField(max_length=200)
-    description = models.TextField(null=True)
-    help = models.TextField(null=True)
-    startNode = models.ForeignKey('FSMNode', related_name='+', null=True)
+    description = models.TextField(null=True, blank=True)
+    help = models.TextField(null=True, blank=True)
```

Comment:
Yes, but it is an old code and this field is used a lot in project - I think it is a refactoring task and in current task we should not do this.
There are a lot of things in the core application to refactor.